tgui/packages/tgui/interfaces/WizardApprenticeContract.js

```diff
@@ -0,0 +1,59 @@
+import { useBackend } from '../backend';
+import { Button, Box, LabeledList, Section, Divider, Fragment } from '../components';
```

Comment:
```suggestion
import { Button, Box, LabeledList, Section, Fragment } from '../components';
```
Pretty sure this is unused now

Reply:
This change too